index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet"> 
+  <link rel="stylesheet" href="./css/style.css">
+  <title>My Portfolio - Squeleton</title>
+</head>
+<body>
+  <nav class="toolbar">
+    <nav class="welcome">
+      <h2>Welcome</h2>
+    </nav>
+    <nav class="dropbox">
+      <a href="#"><img id="drp-img" src="./images/dropbox.png" alt="dropbox"></a>
+    </nav>
+  </nav>
+  <nav class="headline">
+      <nav class="primary-text">
+        <p class="name">
+          Hey there. I’m Mariane
+        </p>
+        <p class="profession">
+          I’m a software developer
+        </p>
+      </nav>
+      <nav class="supporting-text">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </nav>
+    <nav class="social">
```

Comment:
- [ ] To be more semantic instead of a `<nav></nav>` it'd be better to use `<ul></ul>` instead